chsdi/locale/en/LC_MESSAGES/chsdi.po

```diff
@@ -5786,6 +5786,27 @@ msgstr "Territorial regions"
 msgid "ch.vbs.url"
 msgstr "http://www.vbs.admin.ch/internet/vbs/en/home.html"
 
+msgid "ch.vbs.waldschadenkarte"
+msgstr "Waldschadenkarte - Projektil"
```

Comment:
no, for english german has to be used if I understood right : 
http://www.geocat.ch/geonetwork/srv/eng/catalog.search#/metadata/2ff2299e-55b5-4ec7-bb92-fbb06e7ee422